Tests/Features/CloudKitOperationTests.swift

```diff
@@ -1440,6 +1441,45 @@ class CloudKitOperationDiscoverAllContractsTests: CKTests {
         XCTAssertEqual(errorHandlers.count, 1)
         XCTAssertNotNil(errorHandlers[.InternalError])
     }
+
+    func test__set_prepare_for_retry_handler__with_error_which_retries_with_default_retry_handler() {
+        var userInfosByAddress: [String: TestDiscoverUserInfosOperation.DiscoveredUserInfo]? = .None
+        var userInfosByRecordID: [TestDiscoverUserInfosOperation.RecordID: TestDiscoverUserInfosOperation.DiscoveredUserInfo]? = .None
+
+        var shouldError = true
+        let operation: CloudKitOperation<TestDiscoverUserInfosOperation> =
```

Comment:
For whatever reason, the Swift 2.2 compiler does not - it errors. (Not sure about Swift 3.)